ConsoleApp1/HometaskThree/Extention.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1.HometaskThree
+{
+    static class Extention
+    {
+        public static void OutputList(this List<int> list)
+        {
+            Console.WriteLine("Your list is :");
+            foreach (var item in list)
+                Console.Write(item + " ");
+            Console.ReadLine();
+        }
+
+        public static void SortDescending(this List<int> list)
+
+        {
+            list.Sort();
+            list.Reverse();
+            list.OutputList();
```

Comment:
This method should not have an output, each method should be responsible for a single thing, and your method is responsible for a) sorting list, b) output list,
output should be called in main program